man/man8/zfs.8

```diff
@@ -3402,6 +3470,38 @@ Give more parsable tab-separated output, without header lines and without arrows
 Display the path's inode change time as the first column of output.
 .RE
 
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fBzfs key -l\fR \fIfilesystem\fR | \fIvolume\fR\fR
+.ad
+.sp .6
+.RS 4n
+Loads a key into ZFS so that the dataset and all of its children that inherit the \fBencryption\fR property may be accessed. The key will be expected in the format and location specified by the \fBkeysource\fR property. Loading a key alone will not automatically mount the dataset. If that functionality is desired, \fBzfs mount\fR will ask for the key and mount the dataset. When a key is loaded the \fBkeystatus\fR property will be set to available.
```

Comment:
> Any reason why we couldn't specify the keysource here (in addition to defaulting to what the keysource property says)? Seems like you might be in a situation where you usually want to get the key in /var/tmp/key but it isn't there right now because you're in an alternate boot environment and it's in /mnt/var/tmp/key.

Sure. Why not? I'll add that in.

> alone is redundant.

>> available

> should be \fBavailable\fR

Will fix